src/wombats/handlers/simulator.clj

```diff
@@ -0,0 +1,40 @@
+(ns wombats.handlers.simulator
+  (:require [io.pedestal.interceptor.helpers :as interceptor]
+            [clojure.spec :as s]
+            [wombats.daos.helpers :as dao]
+            [wombats.handlers.helpers :refer [wombat-error]]))
+
+(def ^:swagger-spec get-simulator-arena-templates
+  {"/api/v1/simulator/templates"
```

Comment:
Keeping in specs until core.spec is implemented into ring swagger https://github.com/metosin/ring-swagger/issues/95